Maui.RevenueCat.InAppBilling/Platforms/Android/Extensions/OwnershipTypeExtensions.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Maui.RevenueCat.InAppBilling.Enums;
+using OwnershipTypeNative = Com.Revenuecat.Purchases.OwnershipType;
```

Comment:
This is really smart way how to handle our tricky situation.
Maybe one improvement. What about creating `Usings.cs` also in Android folder with content:

```
global using OwnershipTypeNative = Com.Revenuecat.Purchases.OwnershipType;
global using PeriodTypeNative = Com.Revenuecat.Purchases.PeriodType;
```

so we can reuse these usings across the Android implementation ?

Reply:
This issue where both the DTO and the Native version are in the same file doesn't actually show up anywhere else in the Android implementation because this extension method handles the conversion.  I think having this trick close to the usage helps with code clarity and self documents, otherwise it turns into some confusing magic.

Reply:
alright, let's keep it this way. In case we would work with Native types in other parts of the library in the future we can decide to use the Using file